lang/en/tool_objectfs.php

```diff
@@ -93,6 +93,7 @@
 $string['settings:aws:region_help'] = 'Amazon S3 API gateway region.';
 $string['settings:aws:base_url'] = 'Base URL';
 $string['settings:aws:base_url_help'] = 'Alternate url for cnames or s3 compatible endpoints. Leave blank for normal S3 use.';
+$string['settings:aws:upgradeneeded'] = 'Please upgrade \'local_aws\' plugin up to 2020051200 version.';
```

Comment:
you should say, "at least". Probably better to say 'to the latest supported version'

Reply:
Set to `'Please upgrade \'local_aws\' plugin to the latest supported version.'`